connect-inject/container_init_test.go

```diff
@@ -294,6 +358,105 @@ consul-k8s connect-init -pod-name=${POD_NAME} -pod-namespace=${POD_NAMESPACE} \
   -bootstrap > /consul/connect-inject/envoy-bootstrap.yaml`,
 			"",
 		},
+		{
+			"whole template, default namespace, tproxy enabled",
+			func(pod *corev1.Pod) *corev1.Pod {
+				pod.Annotations[annotationService] = "web"
+				return pod
+			},
+			Handler{
+				EnableNamespaces:           true,
+				ConsulDestinationNamespace: "default",
+				EnableTransparentProxy:     true,
+			},
+			`/bin/sh -ec 
+export CONSUL_HTTP_ADDR="${HOST_IP}:8500"
+export CONSUL_GRPC_ADDR="${HOST_IP}:8502"
+consul-k8s connect-init -pod-name=${POD_NAME} -pod-namespace=${POD_NAMESPACE} \
+  -consul-service-namespace="default" \
```

Comment:
no changes needed here, just a general comment that its kinda reading weird now that we have `consul-service-namespace` and `namespace` used across the various connect commands now, I *think* they all mean the same thing but obv we cannot change them on the consul side.

Reply:
Yeah, the problem is that some of these are consul cli flags and some are flags for consul-k8s commands. For consul, there is only one namespace (the consul one), whereas in consul-k8s, it could be either consul or k8s namespace and so we need to disambiguate. Not sure if there is a way around it.